docs/GettingStarted.md

```diff
@@ -151,19 +112,6 @@ either ``sinatra`` or ``sinatra/base``:
 
 The tracing extension will be automatically activated.
 
-#### Configure the tracer
-
-Available settings are:
-
-* ``enabled``: define if the ``tracer`` is enabled or not. If set to ``false``, the code is still instrumented
```

Comment:
Are these settings no longer available?

Reply:
They are still available. They were just moved from a Rails sub-section to one of their own.